runbotics/runbotics-scheduler/src/migrations/1732182583076-Guests.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class Guests1732182583076 implements MigrationInterface {
+    name = 'Guests1732182583076'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "guest" DROP CONSTRAINT "fk_guest_user_id"`);
+        await queryRunner.query(`ALTER TABLE "guest" ALTER COLUMN "user_id" DROP NOT NULL`);
```

Comment:
is this intentional?

Reply:
It shouldn't be here, good catch. I discovered that if we don't specify column in entity and make just some kind of relation like ManyToOne, then those columns in that relation will be nullable - which is not explicitly mentioned.